packages/calico/extra/default_profile.yaml

```diff
@@ -0,0 +1,16 @@
+# change the default profile to allow all traffic from and to endpoints
```

Comment:
nit: filename could be just `profile.yaml`

Reply:
Maybe `default_profile.yaml` is descriptive for its current function, what do you think?

Reply:
In either case, one would still open the file to see the content and the content would tell that it is a `default_profile`. I doubt anybody who is looking would just stop at just reading the filename.

Reply:
Done